docs/CEAP_en_US.md

```diff
@@ -0,0 +1,100 @@
+Article 1 - It can use, above the calculated quota for it's State, an extra of:
+1. BRL 1353,04, the Congress Person who's:
```

Comment:
1. In English locale the separator for decimals is a dot, not a comma. The comma is used as the thousands separator (e.g. 1,353.04)
2. It's **congressperson** (one word, no capital letter)
3. Not sure why but the contraction _who's_ (instead of _who is_) doesn't sound so familiar in that specific case (even if it might be grammatically correct)